app/views/session_mailer/grant_access_to_a_user.text.erb

```diff
@@ -1,3 +1,3 @@
 <%= I18n.t('session_mailer.grant_access_to_a_user.body', session_name: @session.name) %>
 
-<%= I18n.t('session_mailer.grant_access_to_a_user.invitation_link', domain: ENV['WEB_URL'], intervention_id: @session.intervention_id, session_id: @session.id) %>
+<%= I18n.t('session_mailer.grant_access_to_a_user.invitation_link', domain: ENV['WEB_URL'], intervention_id: @session.intervention_id, session_id: @session.id, language_code: @session.intervention.language_code) %>
```

Comment:
What do you think about adding a delegator to the session and instead calling `session.intervention.language_code` just call `session.language_code`?